client/app/layout.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
```

Comment:
Sorry for leaving extra comments after the first review:

Can we already set title to "Stagnum" and description to "A pool-based playback handler for Spotify"?